gazeta-medica.csl

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" page-range-format="minimal" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
```

Comment:
@POBrien333, maybe we should remove the default-locale, even if you didn't test the style in both languages?

http://www.gazetamedica.pt/index.php/gazeta/about/submissions:

> LANGUAGE
> 
> The articles can be submitted in Portuguese or English.

Reply:
Good spot!